verilog/tools/ls/verilog_ls.cc

```diff
@@ -85,7 +85,7 @@ static void SendDiagnostics(const std::string &uri,
 }
 
 int main(int argc, char *argv[]) {
-  const auto file_args = verible::InitCommandLine(argv[0], &argc, &argv);
+  verible::InitCommandLine(argv[0], &argc, &argv);
```

Comment:
Looks ok, or alternatively, we could issue a diagnostic saying that this tool expects no positional arguments.

Reply:
Nobody will ever see the output - the language server is called from some other program.